### PR TITLE
Selftests: Add tests with required minimum CPU.

### PR DESCRIPTION
**Pull request by dawid-sabat**
The idea here is to add a bunch of tests that passes but require various minimum CPU to run. 
This can be useful in development to check if we detect CPU correctly in OpenDCDiag.

### CODE REVIEW

File: bats/sanity-check/35-cpu-specific.bats [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+function setup_file() {
+    setup_cpu_features_tests
+}
+
+function run_min_cpu_test() {
+    local test=$1
+
+    declare -A yamldump
+    sandstone_selftest --beta -e $test
```
Comment (thiagomacieira) [inline bats/sanity-check/35-cpu-specific.bats]: They're all production; `--beta` isn't required.

**Review by thiagomacieira — CHANGES_REQUESTED** [review]
